repair_tests/preview_repair_test.py

```diff
@@ -80,9 +83,18 @@ def test_preview(self):
         # ...and everything should be in sync
         result = node1.repair(options=['ks', '--preview'])
         assert "Previewed data was in sync" in result.stdout
+        # data is repaird, previewFailure metric should remain same
+        assert preivew_failure_count(node1) == 2
 
         result = node1.repair(options=['ks', '--preview', '--full'])
         assert "Previewed data was in sync" in result.stdout
+        assert preivew_failure_count(node1) == 2
 
         result = node1.repair(options=['ks', '--validate'])
         assert "Repaired data is in sync" in result.stdout
+
```

Comment:
Maybe we could finish the test verifying the value of the metric in the other two nodes:
```python
        assert preview_failure_count(node2) == 0
        assert preview_failure_count(node3) == 0
```